### PR TITLE
Add [Instagram] badge

### PR DESCRIPTION
**Pull request by PyvesB**
See  #2120.

### CODE REVIEW

**Comment by PyvesB** [comment]
Result on the review app:

File: services/instagram/instagram.service.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+const Joi = require('@hapi/joi')
+const { metric } = require('../text-formatters')
+const { BaseJsonService } = require('..')
+
+const instagramSchema = Joi.object({
+  graphql: Joi.object({
+    user: Joi.object({
+      edge_followed_by: Joi.object({
+        count: Joi.number()
```
Comment (calebcartwright) [inline services/instagram/instagram.service.js]: Could the `nonNegativeInteger` helper be used here?

Comment (PyvesB) [reply services/instagram/instagram.service.js]: Good idea!

**Comment by PyvesB** [comment]
Do not merge: we've got a problem which I've documented on the linked issue:
https://github.com/badges/shields/issues/2120#issuecomment-554675754

**Pull request closed without merging**